CONTRIBUTING.md

```diff
@@ -39,10 +39,35 @@ contributors a chance to resolve the vulnerability and issue a new release prior
 to any public exposure; this helps protect Apigility users, and provides them
 with a chance to upgrade and/or update in order to protect their applications.
 
-For sensitive email communications, please use 
+For sensitive email communications, please use
 [our PGP key](http://framework.zend.com/zf-security-pgp-key.asc).
 
-## RUNNING TESTS
+## DOCKER FOR DEVELOPMENT
+
+This library requires a running instance of Mongo in order to run and pass
+the unit tests.  It is not expected for each developer to configure their
+individual machine to match this environment so Docker is provided.
+
+### RUNNING DOCKER-COMPOSE
+
+You will need docker-compose installed on your machine.
+Docker supports many PHP versions.  You may specify the version of PHP you
+want (default:7.2) to build against when running `docker-compose` and flag XDEBUG to
+install (1) or not (default:0).
+
+Next, from the root directory of this project, run
+
+```
+docker-compose build --build-arg PHP_VERSION=7.2 --build-arg XDEBUG=0 --no-cache php
+```
+
+To connect to the php container for development run
+
+```
+docker-compose run php bash
+```
+
+### RUNNING UNIT TESTS IN THE CONTAINER
 
 First, use [Composer](https://getcomposer.org) to install all dependencies:
```

Comment:
We don't need that anymore as it is installed on docker. Just we need to note that following commands have to be run in docker container (so after connection as described in previous section).

Reply:
But it isn't installed in docker.  Travis runs composer.  A fresh copy of the module and running Docker will not run composer.

Reply:
It __is__ installed in our docker container:
https://github.com/zfcampus/zf-apigility-doctrine/pull/308/files#diff-ed3ef792c56bca0b45e534f8b1444c01R15 :)

Reply:
Hah, sorry, of course, composer is installed but dependencies are not :) I red this paragraph wrong.